openshift/mlflow-experiment-buildconfig.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-bc
+
+parameters:
+- description: Application name
+  value: 'mlflow-experiment-image'
+  name: APPLICATION_NAME
+  required: true
+- name: GIT_URI
+  value: https://github.com/AICoE/experiment-tracking-template.git
+  required: true
+- name: APP_FILE
+  value: app.py
+  required: true
+  description: Name of the Python application file to be run during the job.
+
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    dockerImageRepository: ${APPLICATION_NAME}
+    tags:
+    - name: latest
+    lookupPolicy:
+      local: true
+
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    resources:
+      limits:
+        memory: 1Gi
```

Comment:
please make this also configurable. 
also add a `cpu` limit with default of `1`